features/raadpleeg-verblijfplaats-met-periode/gba/overzicht-gba.feature

```diff
@@ -0,0 +1,337 @@
+
+  Abstract Scenario: <scenario> (wel vorige en geen volgende verblijfplaats)
+    Gegeven de persoon met burgerservicenummer '000000024' is ingeschreven op adres 'A1' met de volgende gegevens
+    | gemeente van inschrijving (09.10) | datum aanvang adreshouding (10.30) |
+    | 0800                              | 20100818                           |
+    En de persoon is vervolgens ingeschreven op adres 'A2' met de volgende gegevens
+    | gemeente van inschrijving (09.10) | datum aanvang adreshouding (10.30) |
+    | 0800                              | 20160526                           |
+    Als gba verblijfplaatsen wordt gezocht met de volgende parameters
+    | naam                | waarde              |
+    | type                | RaadpleegMetPeriode |
+    | burgerservicenummer | 000000024           |
+    | datumVan            | 2016-09-01          |
+    | datumTot            | 2020-01-01          |
+    Dan heeft de response verblijfplaatsen met de volgende gegevens
+    | adresseerbaarObjectIdentificatie | straat      | datumAanvangAdreshouding |
+    | 0800010000000002                 | Luttestraat | 20160526                 |
+
+    Voorbeelden:
+    | datum van  | scenario                                                  |
+    | 2016-05-26 | De gevraagde periode begint op datum aanvang adreshouding |
+    | 2016-09-01 | De gevraagde periode begint na datum aanvang adreshouding |
+    # 24 |-A1-|--A2--  |-A1-|--A2--
+    #         |---|           |---|
+
+  Abstract Scenario: <scenario> (geen vorige verblijfplaats en wel volgende verblijfplaats)
+    Gegeven de persoon met burgerservicenummer '000000024' is ingeschreven op adres 'A1' met de volgende gegevens
+    | gemeente van inschrijving (09.10) | datum aanvang adreshouding (10.30) |
+    | 0800                              | 20100818                           |
+    En de persoon is vervolgens ingeschreven op adres 'A2' met de volgende gegevens
+    | gemeente van inschrijving (09.10) | datum aanvang adreshouding (10.30) |
+    | 0800                              | 20160526                           |
+    Als gba verblijfplaatsen wordt gezocht met de volgende parameters
+    | naam                | waarde              |
+    | type                | RaadpleegMetPeriode |
+    | burgerservicenummer | 000000024           |
+    | datumVan            | 2010-01-01          |
+    | datumTot            | <datum tot>         |
+    Dan heeft de response verblijfplaatsen met de volgende gegevens
+    | adresseerbaarObjectIdentificatie | straat | datumAanvangAdreshouding | datumAanvangVolgendeAdreshouding |
+    | 0800010000000001                 | Laan   | 20100818                 | 20160526                         |
+
+    Voorbeelden:
+    | datum tot  | scenario                                                                                        |
+    | 2016-05-25 | datumTot van de gevraagde periode is gelijk aan de dag vóór datum aanvang volgende adreshouding |
```

Comment:
hier is juist/ook "datumTot van de gevraagde periode is gelijk aan de datum aanvang volgende adreshouding", dus 2016-05-26, interessant. Dat is de grenswaarde.

Reply:
waarde datumTot aangepast naar datum aanvang volgende adreshouding